src/Device/Driver/XCVario.cpp

```diff
@@ -31,33 +31,54 @@ Copyright_License {
 #include "NMEA/InputLine.hpp"
 #include "util/Clamp.hpp"
 #include "Atmosphere/Pressure.hpp"
+#include "Operation/Operation.hpp"
 #include <math.h>
 
+#define XCV_VERSION_UNKNOWN 0  // as long device has not yet answered and no timeout
+#define XCV_VERSION_1  1       // exchange legacy format with fractional ballast in $PXCV plus CAI302 !g command
+#define XCV_VERSION_2  2       // exchange absolute ballast in liters by !xc[s|v] messages
```

Comment:
Don't use macros. Either use `constexpr` variables, or in this case, an non-strictly-typed `enum` is the appropriate choice. And it shouldn't be global, but be declared inside `class XVCDevice` to avoid namespace clashes.

Reply:
Will put this into an enum inside class XCVDevice, sure.